### ISSUE TITLE
Dismiss Button on Order Screen Is Read Back Incorrectly When Using TalkBack

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-android#7407]
**Describe the bug**
Similar to the dismiss issue on the feedback screen the symbol for dismissing is an X and TalkBack reads this back as Navigate Up button, perhaps the symbol could be changed to a back button like in other parts of the app?

**To Reproduce**
1. Go to 'Orders`
2. Click on 'Product'
3. Scroll down to 'Add Note'
4. Select Add Note
5. Toggle On TalkBack
6. Select Dismiss Button
7. Notice how it reads as `Navigate Up`

**Screenshots**
<img src="https://user-images.githubusercontent.com/36495451/190724749-4e16cfd1-008e-4fbb-ac2b-f4f93b0e8a21.jpg" width="300"/>

**Expected behavior**
This button should be more clear as what it is describing, I imagine navigate up would be confusing to users, maybe change it to "Dismiss" or "close"

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Samsung Galaxy A51
- Android version: 12
- WooCommerce Android version: 10.1

**Comment by shiki** [comment]
I think we intentionally left "Navigate Up" as is since it's expected. But I can't find the p2 for it. 

Do you remember, @Ecarrion @nbradbury @wzieba or @ThomazFB? Do we need to change this? I'm not sure what the standards in Android are.

**Comment by nbradbury** [comment]
@shiki I don't recall this being an intentional decision. Android automatically sets the TalkBack wording to "Navigate up" on the back button. We can probably change this, but I'd prefer to leave it as-is since it's expected.

**Comment by JB184351** [comment]
Oh that's good to know, I'll keep that in mind in the future 😁

**Comment by shiki** [comment]
@nbradbury Yeah, I agree. Let's keep it as is.

I appreciate the report, @JB184351!

**Comment by JB184351** [comment]
> @nbradbury Yeah, I agree. Let's keep it as is.
> 
> 
> 
> I appreciate the report, @JB184351! 

No problem, just didn't know what was expected to work with TalkBack, just the wording felt confusing to me sometimes, depending on the screen I was viewing but if this is consistent throughout the OS makes sense to keep it that way.
